### ISSUE TITLE
Crontab as init system

### ISSUE THREAD

**Issue by lmolent** [A3sal0n/FalconGate#16]
I think is better using systemd to start and stop FalconGate:

```
root@falcongate:~# cat /etc/systemd/system/falcongate.service
[Unit]
Description=FalconGate Server
After=dnsmasq.service
After=nginx.service

[Service]
User=root
WorkingDirectory=/opt/FalconGate
ExecStart=/usr/bin/python falcongate.py
Restart=always

[Install]
WantedBy=multi-user.target
```

**Comment by A3sal0n** [comment]
Thanks for this suggestion @lmolent!

I will proceed to test and implement if it works without issues in RPi and BPi.

**Comment by A3sal0n** [comment]
Enhancement implemented in latest commit (ab7cd45).

Thanks for your suggestion!
